### PR TITLE
overproduction bugfix 

### PR DESCRIPTION
**Pull request by flohump**
**Comment from @bodirsky** 
I finished the bugfix. It now includes substitution between different types of oils and oilcakes, an incentive to use excess oil and ethanol as biofuel, and quality adjustment costs to account for different qualities of oils (e.g. rapeseed being more valuable than palmoil).

I had to include an additional calibration cost factor to incentivize the production of certain types of oils to meet the historical baseline. 
This indicates (again) that our prices and costs are really poor. Oilpalm seems very competitive and has to be punished (with additional costs of 600 USD/ton, so almost doubling the price), while "rapeseed and other oils" has to be subsidized heavily (3000 USD per ton) in order to be produced. I separated reasonable quality adjustments and calibration costs so we can eventually remove the calibration costs when our costs structure is improved. Even after calibration, it doesnt 100% match the baseline, but almost (a bit more oilpalm production at the costs of rapeseed).

### CODE REVIEW

File: modules/60_bioenergy/1stgen_priced_dec18/equations.gms [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+*' is mainly driven by population and GDP.
+
+q60_res_2ndgenBE(i2) ..
+  sum(kres, v60_2ndgen_bioenergy_dem_residues(i2,kres))
+  =g=
+  sum(ct,f60_res_2ndgenBE_dem(ct,i2,"%c60_res_2ndgenBE_dem%"));
+
+*' Finally, an incentive is provided for the production of 1st generation of
+*' bioenergy from oils and ethanol even beyond the exogeneous minimum demand.
+*' The incentive is kept low, but should provide a more realistic substitution
```
Comment (dklein-pik) [inline modules/60_bioenergy/1stgen_priced_dec18/equations.gms]: Do you really want to use these two words "substitution overproduction"? Or rather only one of them?

File: modules/60_bioenergy/1stgen_priced_dec18/equations.gms [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+*' overproduction from couple products.
+
+q60_bioenergy_incentive(i2).. vm_bioenergy_utility(i2)
+          =e= sum(k1st60, vm_dem_bioen(i2,k1st60) * (-300));
```
Comment (dklein-pik) [inline modules/60_bioenergy/1stgen_priced_dec18/equations.gms]: Could you please mention the -300 in the text and add a unit?

**Comment by dklein-pik** [comment]
Please see my comments at the bottom of 
modules/60_bioenergy/1stgen_priced_dec18/equations.gms

File: modules/60_bioenergy/1stgen_priced_dec18/equations.gms (hunk 1 above)
Comment (flohump) [reply modules/60_bioenergy/1stgen_priced_dec18/equations.gms]: I removed the word "substitution".

File: modules/60_bioenergy/1stgen_priced_dec18/equations.gms (hunk 2 above)
Comment (flohump) [reply modules/60_bioenergy/1stgen_priced_dec18/equations.gms]: I mention the 300 incl unit in the text.

**Comment by dklein-pik** [comment]
As far as I can see everything looks fine except two little things regarding the last equation in

**Pull request merged**